styles/layout.css

```diff
@@ -1,14 +1,38 @@
 body {
-    padding: 0;
-    margin: 0;
-    font-family: sans-serif;
-    background-color: #350a3d;
-    color: #eeeec5;
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+  background-color: #350a3d;
+  color: #eeeec5;
 }
 
 .container {
-    width: 100%;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-}
\ No newline at end of file
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/*
+ * TODO: figure out how to adjust mouse coordinates for object-fit: contain
```

Comment:
No action at present, just adding some details for future reference:

For mobile, the screen is usually too small for the 500x700 px image so I'd like to use this CSS to shrink the canvas. However, this messes up `p5.mouseX/mouseY` so I'll have to compute it myself. Since this applies to multiple sketches, I'll have to make a common utility function for this. Out of scope for today.